src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,49 @@
+import { useState, useEffect } from 'react';
+import { Loader } from '../components/Loader/Loader';
+import { PeopleTable } from '../components/PeopleTable/PeopleTable';
+import { Person } from '../types/Person';
+import { getPeople } from '../api';
+
+export const PeoplePage = () => {
+  const [tablePeople, setTablePeople] = useState<Person[]>([]);
+  const [hasLoading, setHasloading] = useState<boolean>(true);
```

Comment:
There is a typo in the state variable name `setHasloading`. It should be `setHasLoading` to match the camelCase convention and the variable `hasLoading`.